src/components/sidebar/Sidebar.jsx

```diff
@@ -85,6 +84,17 @@ const Sidebar = ({ menuOpened, setMenuOpened }) => (
         <li className="nav-item">
           <ThemeToggler />
         </li>
+        <li className="nav-item">
+          <NavLink
+            to="/detail-page"
+            onClick={() => {
+              setMenuOpened();
+            }}
+          /* eslint-disable-next-line */
+          >
+          Detail Pages
+          </NavLink>
+        </li>
```

Comment:
[question] we won't need this link in navigation when Michael implements doctor UI it will be linked from there, correct?
I know why you might have it in there so we can easily access it, Thank you for that 😇

Reply:
No, we won't need it, all doctors will be displayed in the "Doctors" tab (Michael's task), then when the user clicks over a user the App will display the details of that specific doctor, so basically, doctors' cards will be the link to the details page.

Reply:
Great, then! so we don't have to properly style it for now 👍🏻

Reply:
Exactly, that’s the reason why is not styled!